packages/account/src/Sections/Security/ClosingAccount/account-has-balance.jsx

```diff
@@ -270,6 +284,55 @@ const AccountHasPendingConditions = ({
                         ))}
                     </Wrapper>
                 )}
+                {!!ctrader_open_positions.length && (
+                    <Wrapper title={localize('Please close your positions in the following Deriv cTrader account(s):')}>
```

Comment:
you can use `Localize` and replace Deriv cTrader with the `constant` value you've defined in cfd-text.

Reply:
Done